config/settings.py

```diff
@@ -161,3 +161,4 @@
 EMAIL_USE_TLS = True
 
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+X_FRAME_OPTIONS = 'ALLOW-FROM https://127.0.0.1/'
```

Comment:
What is this and how will this work upon deployment (when the url is not 127...)